examples/storage/project/versions.tf

```diff
@@ -20,5 +20,5 @@ terraform {
 }
 
 provider "google" {
-  version = "~> 3.36.0"
+  version = "~> 3.0"
```

Comment:
We usually keep these pinned to minor versions to prevent tests from breaking too often.

Reply:
Sounds good. I updated the version to .41 because in .38 the uniform bucket policy field was introduced.

Reply:
@umairidris could we also update [required_providers](https://github.com/terraform-google-modules/terraform-google-log-export/blob/master/modules/storage/versions.tf) accordingly?

Reply:
Good catch, done. Do I need to make this PR a breaking release as well?

Reply:
We had some troubles with https://github.com/terraform-google-modules/terraform-google-project-factory/issues/462 so I would say its safer to do a breaking release. We can bundle it together with https://github.com/terraform-google-modules/terraform-google-log-export/pull/64 ?

Reply:
Sounds good.